backend/migrations/000034_update_community_catergory.down.sql

```diff
@@ -0,0 +1,9 @@
+DELETE FROM community_types WHERE key='nft';
+DELETE FROM community_types WHERE key='collector';
+
+BEGIN;
+ALTER TYPE community_type RENAME VALUE 'collector' TO 'service';
+END TRANSACTION;
+COMMIT;
```

Comment:
NFT is not being removed from the `community_type` I think it's not necessary to address that in the enum type